src/es-module-shims.js

```diff
@@ -1,5 +1,5 @@
 import { baseUrl as pageBaseUrl, parseImportMap, resolveImportMap, createBlob } from './common.js';
-import { analyzeModuleSyntax } from './lexer.js';
+import analyzeModuleSyntax from '../node_modules/es-module-lexer/lexer.js';
```

Comment:
Unfortunately it's not quite this simple as I did actually change the API for dynamic import in the process of splitting out the lexer into its own package.

For `import('asdf')`, instead of the values being `import.s` reflecting the start of the `import(` statement and `import.d` being the start of the `'asdf` part, I switched it around so that `source.slice(import.s, import.e)` is the `'asdf` specifier and `import.d` is the `import(` statement part.

Reply:
Updated to the new syntax. Added a test for dynamic import as well :) Please double check